sfge/src/test/java/com/salesforce/rules/usewithsharingondatabaseoperation/SharingPolicyDatabaseOperationInOtherClassMethodTest.java

```diff
@@ -0,0 +1,133 @@
+package com.salesforce.rules.usewithsharingondatabaseoperation;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+/**
+ * tests MyClass instantiating OtherClass and calling DatabaseClass#doDbOp which contains a database
+ * operation.
+ */
+public class SharingPolicyDatabaseOperationInOtherClassMethodTest
+        extends UseWithSharingOnDatabaseOperationTest {
+    // spotless:off
+    private static final String MAIN_CLASS =
+        "public %s class " + MY_CLASS + " {\n" +
+            "void foo(Account a) {\n" +
+            "   OtherClass sub = new OtherClass();\n" +
+            "   sub.doDbOp(a);\n" +
+            "}\n" +
+        "}\n";
+
+    private static final String OTHER_CLASS =
+        "public %s class OtherClass {\n" +
+            "   void doDbOp(Account a) {\n" +
+            "       %s;\n" +
+            "   }\n" +
+            "}\n";
+    // spotless:on
+
+    private static final int SINK_LINE = 3;
+
+    // MAIN: WITH SHARING
```

Comment:
These tests also seem like they could be consolidated. Add a `TODO` for it, and we'll get to it in the fullness of time.